### DISCUSSION TITLE
Expansion Item how to remove click event to dropdown icon

### DISCUSSION THREAD

**Discussion by akashramtohul4** [quasarframework/quasar#10938] [General - Components / Directives / etc] [answered]
Hello friends ,

Is it possible to remove the click event or even remove the icon on the drowdown icon in expansion item 
![image](https://user-images.githubusercontent.com/84211129/136671197-953dfff5-9b06-476c-adca-7d0ca65a2c0c.png)

can someone please help
thanks

**Comment by MilosPaunovic** [comment]
Is there any particular reason why you would want to remove expansion icon and action from expansion item? Why not just use regular item?

**Reply by akashramtohul4** [reply]
@MilosPaunovic  infect I have a toggle that is doing the same thing 
![image](https://user-images.githubusercontent.com/84211129/136673147-40ec1159-d808-46c2-9694-399631418cb5.png)
 but when i select on the toggle it makes the expansion icon void

**Comment by rstoenescu** [comment, marked as answer]
`<q-expansion-item expand-icon-class="hidden" expand-icon-toggle>`
